js/signUp.js

```diff
@@ -13,18 +13,41 @@ const passwordAgainErrorEl = document.querySelector('.password-again.error-msg')
 // 비밀번호 유효성 검사식
 const passwordPattern = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
 
-const checkEmailInput = () => {
+const checkEmailInput = async function () {
   if (emailInputEl.value == '') {  // 만약 인풋 태그가 비어있다면
     validateInput(emailErrorEl, '이메일을 입력해주세요.', emailErrorEl);
   } else if (emailPattern.test(emailInputEl.value) == false) {  // 이메일 유효성 검사
     validateInput(emailErrorEl, '올바른 이메일 주소가 아닙니다.', emailErrorEl);
-  } else if (emailInputEl.value === accountInfo.codeit.id) {
-    validateInput(emailErrorEl, '이미 사용중인 이메일입니다.', emailErrorEl);
   } else {
-    validateInput(emailErrorEl, '', emailErrorEl, 'var(--gray-three)');
+    const isDuplicate = await checkEmailDuplicate();
+    if (!isDuplicate) {
+      validateInput(emailErrorEl, '이미 사용중인 이메일입니다.', emailErrorEl);
+    } else {
+      validateInput(emailErrorEl, '', emailErrorEl, 'var(--gray-three)');
+    }
   }
 }
 
+const checkEmailDuplicate = async function () {
+  const signUpEmail = { email: emailInputEl.value };
```

Comment:
위에서는 `email`로 정의해 주시고
```suggestion
  const email =  emailInputEl.value
```

아래에서 단축문법으로 간단히 넣어주시는 방법도 있습니다!  ([참고 링크](https://pro-self-studier.tistory.com/33))
```js
body: JSON.stringify({ email }),
```